src/config/normalize.js

```diff
@@ -1,12 +1,14 @@
-import { Dimensions } from 'react-native';
+import { Dimensions, PixelRatio } from 'react-native';
 
-// Default guideline sizes are based on standard ~6.12" screen mobile device (iPhone 14 Pro)
 const guidelineBaseWidth = 393;
+const guidelineBaseHeight = 852;
 
 export const normalize = (size, factor = 1) => {
-  const { width, height } = Dimensions.get('window');
-  const shortDimension = width < height ? width : height;
-  const scale = (shortDimension / guidelineBaseWidth) * size;
+   const { width, height } = Dimensions.get('window');
+   const widthScale = width / guidelineBaseWidth;
+   const heightScale = height / guidelineBaseHeight;
+   const scale = Math.min(widthScale, heightScale); // Takes the smaller ratio to avoid excess.
 
-  return size + (scale - size) * factor;
-};
+   const newSize = size * scale;
+   return PixelRatio.roundToNearestPixel(newSize * factor);
```

Comment:
We do not need this `newSize` variable, because we just use it once. Its name is also not very helpful. We can use `size * scale * factor` directly in the `return`. This even helps readability of the code, as one does not need to jump to the different row in order to know, what calculation is done.

Reply:
Thanks, adjusted in the newest commit: https://github.com/smart-village-solutions/smart-village-app-app/pull/1127/commits/33d36bc8117f6958b9f362b64f91ec57a05d4cb1